misc/assets/anonymization.svg

```diff
@@ -33,20 +33,26 @@
   <g transform="translate(300 100)">
     <use href="#model-wide" />
     <text class="caption">AttachmentNormalization</text>
-    <path class="relation from-many to-one" d="M 125,0 213,0 213,0 213,100"/>
+    <path class="relation from-many to-one" d="M 125,0 213,0"/>
   </g>
   <g transform="translate(300 200)">
     <use href="#model-wide" />
     <text class="caption">AttachmentRecognition</text>
-    <path class="relation from-many to-one" d="M 125,0 300,0" />
+    <path class="relation from-many to-one" d="M 125,0 213,0" />
   </g>
   <g transform="translate(300 300)">
     <use href="#model-wide" />
     <text class="caption">AttachmentAnonymization</text>
-    <path class="relation from-many to-one" d="M 125,0 213,0 213,0 213,-100" />
+    <path class="relation from-many to-one" d="M 125,0 213,0" />
   </g>
-  <g transform="translate(700 200)">
-    <use href="#external" class="wide" />
+  <g transform="translate(300 400)">
+    <use href="#model-wide" />
+    <text class="caption">AttachmentFinalization</text>
+    <path class="relation from-many to-one" d="M 125,0 213,0" />
+  </g>
+  <g transform="translate(700 250)">
+    <use href="#external" />
     <text class="caption">Attachment</text>
+    <path class="relation" d="M -100,0 -187,0 -187,150 -187,-150" />
```

Comment:
Nepozeraj sa na to SVG ako na obrazok, ale ako na zapis modelu. Radsej mat ciary definovane iba v `<g>` odkial "vychadzaju" (tj. v ktorych modeloch su definovane) a nesekat ich takto na kusky. Lebo potom bude to SVG neprehladne. Ci z nejakeho dovodu vadi, ze sa tie ciary prekryvaju?

Pre jednoduchost sa nemusime babrat s tym, aby `Attachment` bol vertikalne v strede. Kludne ho daj do rovnakej vysky ako `AttachmentNormalization`

Reply:
:ok: